src/client/client.tsx

```diff
@@ -0,0 +1,7 @@
+import EventsJSON from "../data/events.json";
+
+export function fetchEvent(id: Number): Promise<Event> {
```

Comment:
`Event` type won't work for the JSON objects, so as a temporary fix I think you can just do:

```suggestion
export function fetchEvent(id: Number): Promise<any> {
```

Reply:
Wait I think it's better to keep this as Event, and cast the object as an Event inside the function-- it should make it easier to call fetchEvent as a helper

Reply:
The wrong Event class was being used here, fixed